build.gradle

```diff
@@ -85,6 +88,10 @@ tasks.withType(Test).configureEach {
     }
 }
 
+java {
+    sourceCompatibility = JavaVersion.VERSION_11
+}
+
 tasks.named('bootJar') { enabled = false }
```

Comment:
Is bootJar forced to false in the 6.x grails plugin gradle plugin?  Is this even needed?

Reply:
No, I'll remove it :+1: